Dockerfile

```diff
@@ -85,14 +85,15 @@ RUN --mount=type=bind,target=./requirements/local.txt,src=./requirements/local.t
     --mount=type=cache,target=/root/.cache/pip \
     pip install -r requirements/local.txt
 
-COPY --chown=superset:superset --from=superset-node /app/superset/static/assets superset/static/assets
-## Lastly, let's install superset itself
-COPY --chown=superset:superset superset superset
 RUN --mount=type=cache,target=/root/.cache/pip \
     pip install -e . \
     && flask fab babel-compile --target superset/translations \
     && chown -R superset:superset superset/translations
 
+COPY --chown=superset:superset --from=superset-node /app/superset/static/assets superset/static/assets
```

Comment:
I think moving this AFTER the `pip install` command is going to be a huge win. Currently if you touch any JS file you'll have to re-pip install everything, where now it shouldn't be the case anymore. Frontend PRs/SHAs won't have to import and rebuild the backend.